### ISSUE TITLE
Confusing error message when manually specifying authenticator

### ISSUE THREAD

**Issue by jsha** [letsencrypt/letsencrypt#971]
Steps to reproduce:

```
TMP=`mktemp -d`
letsencrypt --email foo@example.com --server http://localhost:4000/directory  --text --agree-dev-preview --agree-tos -d foo.com --config-dir $TMP --work-dir $TMP --logs-dir  $TMP --dvsni-port 5001 --nginx-server-root $TMP -a nginx  auth 
```

Expected result:

"Failed to initialize Nginx plugin: Could not find nginx.conf"

Actual result:

"Authenticator could not be determined"

**Comment by pde** [comment]
Partially addressed by #1001

**Comment by pde** [comment]
So this has improved; we now say: "The nginx plugin is not working; there may be problems with your existing configuration".  We should mention the logfile or better yet, surface the error.
